io.embrace.sdk/Scripts/Native/Embrace_iOS.cs

```diff
@@ -12,246 +12,266 @@ namespace EmbraceSDK.Internal
     ///  Embrace_iOS uses statically linked methods that are linked into the executable using the DllImportattribute.  For iOS, we have to use “__Internal” as the library name for the attribute [DllImport("__Internal")].  
     ///  When a user makes a call to our Embrace class this is passed on to a method from the provider interface which then calls the external method. 
     /// </summary>
-#if UNITY_IOS || UNITY_TVOS
+#if (UNITY_IOS || UNITY_TVOS)
```

Comment:
Does somebody uses `UNITY_TVOS`? If that's the case, two things about that:
1. The `.xcframework`s from the `embrace-apple-sdk` doesn't include the `tvOS` binaries.
2. The `Embrace Unity iOS Interface/build_xcframework.sh` is not creating a `tvOS` binary.

Reply:
I actually know of no customers offhand who use TVOS at this time. Notable that none of our xcframeworks support it, so we couldn't handle this case anyways. I'll strip out TVOS support just to make sure.

Reply:
Maybe just commenting that until we have proper support in our `.xcframeworks`; it's a matter of testing it mostly on the iOS SDK side